### ISSUE TITLE
NVDA Cannot Access The Context Menu in Microsoft Edge History

### ISSUE THREAD

**Issue by Shah2** [nvaccess/nvda#15454]
### Steps to reproduce:
In Microsoft Edge, open Edge history with Control + h. Select any of the history items with the arrow keys. Press the Application key or Shift + F10 for the context menu.
### Actual behavior:

speak command keys on
ctrl+h
History
heading    level 1  button    History
tool bar    button    Clear browsing data
menu button  collapsed  subMenu    More options
button    Pin history  out of tool bar
All  property page
All  tree view
Today - Saturday, 16 September 2023  1 of 2  level 1
down arrow
grouping
level 2  New Issue · nvaccess/nvda, 10:02, https://github.com/nvaccess/nvda/issues/new?assignees=&labels=&projects=&template=bug_report.md  New Issue · nvaccess/nvda https://github.com/nvaccess/nvda/issues/new?assignees=&labels=&projects=&template=bug_report.md  1 of 17
applications
down arrow
Issues · nvaccess/nvda, 10:00, https://github.com/nvaccess/nvda/issues  Issues · nvaccess/nvda https://github.com/nvaccess/nvda/issues  2 of 17  level 2
up arrow
New Issue · nvaccess/nvda, 10:02, https://github.com/nvaccess/nvda/issues/new?assignees=&labels=&projects=&template=bug_report.md  New Issue · nvaccess/nvda https://github.com/nvaccess/nvda/issues/new?assignees=&labels=&projects=&template=bug_report.md  1 of 17  level 2
down arrow
Issues · nvaccess/nvda, 10:00, https://github.com/nvaccess/nvda/issues  Issues · nvaccess/nvda https://github.com/nvaccess/nvda/issues  2 of 17  level 2
shift+f10
down arrow
NVDA crashes Google Chrome on certain websites · Issue #15451 · nvaccess/nvda, 09:58, https://github.com/nvaccess/nvda/issues/15451  NVDA crashes Google Chrome on certain websites · Issue #15451 · nvaccess/nvda https://github.com/nvaccess/nvda/issues/15451  3 of 17  level 2
down arrow
nvda/projectDocs/issues/githubIssueTemplateExplanationAndExamples.md at ec914517ec3452c51f33f6d2f5df8daea3196cc1 · nvaccess/nvda, 09:50, https://github.com/nvaccess/nvda/blob/ec914517ec3452c51f33f6d2f5df8daea3196cc1/projectDocs/issues/githubIssueTemplateExplanationAndExamples.md  nvda/projectDocs/issues/githubIssueTemplateExplanationAndExamples.md at ec914517ec3452c51f33f6d2f5df8daea3196cc1 · nvaccess/nvda https://github.com/nvaccess/nvda/blob/ec914517ec3452c51f33f6d2f5df8daea3196cc1/projectDocs/issues/githubIssueTemplateExplanationAndExamples.md  4 of 17  level 2
alt+tab
NVDA Speech Viewer  2 of 5
alt+tab
nvda  3 of 5
alt+tab
foobar2000 v2.0  4 of 5
alt+tab
Skype [1]  5 of 5
alt+tab
New Issue · nvaccess/nvda - Personal - Microsoft​ Edge  1 of 5
alt+tab
NVDA Speech Viewer  2 of 5
NVDA Speech Viewer
Show Speech Viewer on Startup  check box  not checked  Alt+s
NVDA+4
speak command keys off

### Expected behavior:

NVDA should read the context menu items.
#### NVDA installed/portable/running from source:
Portable version.
#### NVDA version:
Version: 2023.3beta1 (2023.3.0.29178)
#### Windows version:
Microsoft Windows
Version 22H2 (OS Build 22621.2283)
#### Name and version of other software in use when reproducing the issue:
Microsoft Edge
Version 116.0.1938.81 (Official build) (64-bit)
#### Does the issue still occur after restarting your computer?
Yes.
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes. Same behavior with the installe NVDA 2023.2.
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes.
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Did not run the tool.

**Comment by leonardder** [comment]
It looks like the context menu in this case just does not open. I can also reproduce this with Narrator. This is most definitely a bug in Edge itself.
